### PR TITLE
Implement `acosh` f32 tests

### PR DESCRIPTION
**Pull request by zoddicus**
Issue #1221

<hr>

**Requirements for PR author:**

- [x] All missing test coverage is tracked with "TODO" or `.unimplemented()`.
- [x] New helpers are `/** documented */` and new helper files are found in `helper_index.txt`.
- [x] Test behaves as expected in a WebGPU implementation. (If not passing, explain above.)

**Requirements for [reviewer sign-off](https://github.com/gpuweb/cts/blob/main/docs/reviews.md):**

- [x] Tests are properly located in the test tree.
- [x] [Test descriptions](https://github.com/gpuweb/cts/blob/main/docs/intro/plans.md) allow a reader to "read only the test plans and evaluate coverage completeness", and accurately reflect the test code.
- [x] Tests provide complete coverage (including validation control cases). **Missing coverage MUST be covered by TODOs.**
- [x] Helpers and types promote readability and maintainability.

When landing this PR, be sure to make any necessary issue status updates.

### CODE REVIEW

**Comment by zoddicus** [comment]
This includes the alternative from Vulkan CTS, which is discussed in https://github.com/gpuweb/gpuweb/issues/3287

File: src/webgpu/shader/execution/expression/call/builtin/acosh.spec.ts [hunk 1]
```diff
@@ -31,7 +36,14 @@ g.test('f32')
   .params(u =>
     u.combine('inputSource', allInputSources).combine('vectorize', [undefined, 2, 3, 4] as const)
   )
-  .unimplemented();
+  .fn(async t => {
+    const makeCase = (n: number): Case => {
+      return makeUnaryF32IntervalCase(n, ...acoshIntervals);
```
Comment (zoddicus) [inline src/webgpu/shader/execution/expression/call/builtin/acosh.spec.ts:41]: Is having `...acoshIntervals` better than `acoshAlternativeInterval, acoshPrimaryInterval`?
Explicitly stating the functions is how other tests currently handle this, but I considering converting them over to having an array of functions like this, so the tests don't have to iterate all of the options.

Comment (dneto0) [reply src/webgpu/shader/execution/expression/call/builtin/acosh.spec.ts:41]: This looks fine. I'd say coders choice, but this is slightly more preferable, IMHO.

File: src/webgpu/shader/execution/expression/call/builtin/acosh.spec.ts [hunk 2]
```diff
@@ -31,7 +36,14 @@ g.test('f32')
...
+    };
+
+    const cases = fullF32Range().map(makeCase);
```
Comment (dneto0) [inline src/webgpu/shader/execution/expression/call/builtin/acosh.spec.ts:44]: I'd feel better if there was a relatively dense set of points around the "hard" part of the function: at x=1 and just above 1.

**Review by dneto0 — CHANGES_REQUESTED** [review]
I'd like more attention shown to the area  x in [1, 1+small ]

File: src/webgpu/shader/execution/expression/call/builtin/acosh.spec.ts (hunk 2 above)
Comment (zoddicus) [reply src/webgpu/shader/execution/expression/call/builtin/acosh.spec.ts]: Added some additional test cases near x = 1

Comment (zoddicus) [reply src/webgpu/shader/execution/expression/call/builtin/acosh.spec.ts]: I used a biased interval of [1, 2] with 100 samples for the additional test cases, I can easily change the bounds if you want.

**Review by dj2 — APPROVED** [review]

**Review by dneto0 — APPROVED** [review]
Thanks!

**Pull request merged**